sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -1058,7 +1057,7 @@ def begin_analyze_actions(
             entire batch. For example, use "en" for English; "es" for Spanish etc.
             If not set, uses "en" for English as default. Per-document language will
             take precedence over whole batch language. See https://aka.ms/talangs for
```

Comment:
I was curious and the link doesn't seem to be taking us to the place we want. I know it is owned by the service though.
We can contact them and get it updated to https://docs.microsoft.com/azure/cognitive-services/language-service/language-detection/language-support

Reply:
or just remove the link

Reply:
lemme follow up with the service team, I think there's already a thread for this.